api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,47 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="integer" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="additional_detail" type="varchar(255)">
+				<constraints nullable="true"/>
+			</column>
+			<column name="previous_version" type="org.openmrs.Condition"></column>
+			<column name="condition" type="varchar(255)" />
+			<column name="clinical_status" type="varchar(50)" >
+				<constraints nullable="false" />
+			</column>
+			<column name="verification_status" type="varchar(50)" >
+				<constraints nullable="false"/>
+			</column>
+			<column name="onset_date" type="datetime" />
+			<column name="date_created" type="datetime">
+				<constraints nullable="false" />
+			</column>
+			<column name="voided" type="boolean">
+				<constraints nullable="false" />
+			</column>
+			<column name="date_voided" type="datetime" />
+			<column name="void_reason" type="varchar(255)" />
+			<column name="uuid" type="varchar(38)">
+				<constraints unique="true" />
+			</column>
+			<column name="creator" type="org.openmrs.User">
```

Comment:
creator should also be of type int
Liquibase takes types that appear in the database. Not in the java side of things.

Reply:
Oh. . . that is an eye opener! Thank you!